asyncapi-cli/src/main/java/io/ballerina/asyncapi/cmd/websockets/BallerinaToAsyncAPIGenerator.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```
Shall we fix it in other places as well?